play/change-maker/js/app.js

```diff
@@ -1,44 +1,82 @@
-// coins objects array to hold information about the coins for the game to use
+/**
+* coins objects array to hold information about the coins for the game to use
+* The image source points to the change-maker directory instead of relative path so keep the path static and local
+**/
 const COINS = [
     {
         name: "Quarter",
         value: 25,
-        img: "https://upload.wikimedia.org/wikipedia/commons/4/44/2014_ATB_Quarter_Obv.png"
+        img: "quarter.png",
+        description : "The value is $0.25"
     },
     {
         name: "Dime",
         value: 10,
-        img: "https://upload.wikimedia.org/wikipedia/commons/thumb/d/dd/2017-D_Roosevelt_dime_obverse_transparent.png/440px-2017-D_Roosevelt_dime_obverse_transparent.png"
+        img: "dime.png",
+        description : "The value is $0.10"
     },
     {
-        name: "Nicket",
+        name: "Nickel",
         value: 5,
-        img: "https://i.dlpng.com/static/png/94407_thumb.png"
+        img: "nickle.png",
+        description : "The value is $0.05"
     },
     {
         name: "Penny",
         value: 1,
-        img: "https://i.dlpng.com/static/png/303295_preview.png"
+        img: "penny.png",
+        description : "The value is $0.01"
     }
 ];
 
 // global variables to hold game state
 let score = 0;
 let targetValue = 0;
 let currentValue = 0;
+let booleanTimer = false;
+let timeLeft = 30;
 
 /**
  * Initalize the UI on script load.
  */
-(function initUI() {
+function initUI() {
+    const model = document.querySelector(".modal");
+    model.style.display = "none";
     // initialize coin behavior UI stuff
     enableCounterDropDetection();
     createCoinElements();
 
     // get a new target value for the user as their target
     targetValue = getRandomNumber();
     updateReadout();
-})();
+
+    // bind bootstrap popper.js to the tooltips
+    $(`[data-toggle="tooltip"]`).tooltip({ delay: { show: 300, hide: 150 }, html: true, placement: "top", animation: true });
+
+    if(booleanTimer) {
+        document.getElementById("timer").style.display = "inline-block";
+        setTimer;
```

Comment:
Is this supposed to be a function call or is this something that was forgotten to be removed?

Reply:
setTimer is a variable calling a function so by simply saying setTimer; is really calling the function while setTimer() throws an error.